src/open_inwoner/accounts/backends.py

```diff
@@ -185,4 +189,14 @@ def create_user(self, claims):
             }
         )
 
+        if vestigingsnummer := claims.get("vestigingsnummer", None):
+            self.request.session[KVK_BRANCH_SESSION_VARIABLE] = vestigingsnummer
+            self.request.session.save()
```

Comment:
As discussed, we don't have a helper function for storing the value in the session, and since the functions are quite simple, adding another one doesn't seem to be of much use. Let's continue importing and using `KVK_BRANCH_SESSION_VARIABLE`